engine/dlib/src/test/test_csharp.cpp

```diff
@@ -0,0 +1,51 @@
+#include <stdio.h>
+#include <memory.h>
+
+#define JC_TEST_IMPLEMENTATION
+#include <jc_test/jc_test.h>
+
+#include <dmsdk/dlib/hash.h>
+#include <dmsdk/dlib/configfile.h>
+#include <dlib/configfile.h>
+
+extern "C" int      csRunTestsHash();
+extern "C" uint64_t csHashString64(const char* s);
+
+extern "C" int      csRunTestsConfigFile(HConfigFile config);
```

Comment:
We link towards a NativeAOT static library, in which we expect it to have two public symbols `csRunTestsHash()` and `csRunTestsConfigFile()`